wandb/sdk/lib/apikey.py

```diff
@@ -123,10 +123,8 @@ def prompt_api_key(  # noqa: C901
         )
 
     key = None
-    api_ask = (
-        f"{log_string}: Paste an API key from your profile and hit enter, "
-        "or press ctrl+c to quit"
-    )
+    api_ask_suffix = ", or press ctrl+c to quit" if not jupyter else ""
```

Comment:
I think it will be more readable if you just have 2 complete string for non notebook and notebook cases